docs/running.rst

```diff
@@ -960,21 +964,25 @@ The attributes of this handler are the following:
 
 * ``format``: The syntax of this attribute is the same as of the standard logging facility, except that it adds a couple more performance-specific formatting placeholders:
 
+  - ``check_perf_num_tasks``: The number of tasks assigned to the test.
```

Comment:
This should be `check_num_tasks` and it should belong to the general check log attributes.